index.html

```diff
@@ -13,16 +10,72 @@
   </head>
   <body>
 
+<!--        header            -->
     <header>
-      
-    </header>
+      <section class="left-nav-container">
+
+        <div class="nav-choice-container">
+          <img class="icon" src="./assets/GPS.svg" alt="gps icon">
+          <p class="nav-link">navlink</p>
+        </div>
 
+        <div class="nav-choice-container">
+          <img class="icon" src="./assets/GPS.svg" alt="gps icon">
+          <p class="nav-link">navlink</p>
+        </div>
+
+        <div class="nav-choice-container">
+          <img class="icon" src="./assets/GPS.svg" alt="gps icon">
+          <p class="nav-link">navlink</p>
+        </div>
+
+      </section>
+      <section class="right-nav-container">
+        <div class="user-container">
+          <img class="icon-user" src="./assets/GPS.svg" alt="gps icon">
+          <p class="user">Username</p>
+        </div>
+        <img class="logo" src="./assets/GPS.svg" alt="placeholder">
+      </section>
+    </header>
+<!--          mainsection            -->
     <main>
+      <h3>Popular Sammiches</h2>
+      <section class="card-container">
 
+        <article class="card">
```

Comment:
You are using the `<article class="card">` card quite a bit. This is a good way of laying out you page initially, but what would happen if you needed 20 cards? Or what if the number of your cards is unknown? How could you programmatically solve that problem?